templates/addons/etcd-backup-restore/etcd-backup-restore.yaml

```diff
@@ -80,7 +80,7 @@ data:
             - --cacert=/etc/kubernetes/pki/etcd/ca.crt
             - --cert=/etc/kubernetes/pki/etcd/healthcheck-client.crt
             - --key=/etc/kubernetes/pki/etcd/healthcheck-client.key
-            image: ${ETCDBR_IMAGE}
+            image: ${ETCDBR_IMAGE:docker.io/amoldeodhar/etcdbrctl:withcredsfile}
```

Comment:
Can this also be used with the upstream image? or does it only work with the upstream changes?

Reply:
snapshotting and uploads should work but SSE will definitely not work with the current upstream image

Reply:
Also, given that upstream makes breaking changes often(and they might change the approach we have taken if and when they also incorporate AWS KMS Key support alongwith our SSE-C support), our addon implementation will often be tightly coupled with an upstream image tag

Reply:
I am fine with pinning to a specific version, I just want to make sure we aren't using a fork because we can't use the upstream version unless it has the exact spec we have currently just in case it takes a while to merge/things change and we are stuck on a fork. could we point at the latest supported upstream version without SSE for now and add in the SSE version once it is supported?

Reply:
I've reverted this change and moved it to [this commit](https://github.com/linode/cluster-api-provider-linode/pull/218/commits/1799ebe29cd4c0c2f23923328d861655e2ab1e30) on [PR-218](https://github.com/linode/cluster-api-provider-linode/pull/218). I've tested that we can use that image and the snapshot is uploaded unencrypted.